ibis/backends/datafusion/tests/test_register.py

```diff
@@ -45,6 +44,8 @@ def test_register_batches(conn):
 
 
 def test_register_dataset(conn):
+    import pyarrow.dataset as ds
```

Comment:
same that the comment above. If we are good with these nits getting in here, all good.

Reply:
Definitely related, for the same reason.